### ISSUE TITLE
Not as clear as yours

### ISSUE THREAD

**Issue by Jw-Jn** [otenim/GLCIC-PyTorch#2]
Hi, I think your job is great and I ran your demo, it is quite good. But when I try to train celebA by myself, the result is a little blurry. I only change the batch size, so what other things do you think would have an effect on that?

**Comment by otenim** [comment]
Hi, Jw-Jn.

Sorry for my late reply. How big is the batch size you tried ?

**Comment by Jw-Jn** [comment]
32, is it too large?

**Comment by Jw-Jn** [comment]
Thank you for your reply.
yes, I've used them, but in a different version: numpy==1.14.3 & pyamg==4.0.0, (and I think the version does not matter?) 
Also, I've noticed that after training for many steps, the loss just change a little bit, so is it OK that I minus the default training steps?

**Comment by otenim** [comment]
> Also, I've noticed that after training for many steps, the loss just change a little bit, so is it OK that I minus the default training steps?

It would be OK to change the default setting according to the problems.

>  yes, I've used them, but in a different version: numpy==1.14.3 & pyamg==4.0.0, (and I think the version does not matter?)

If you use pyamg4 the poissonbelnding.py does not work correctly so I recommend you to use pyamg3.
Could you please show me the result images ? I wanna specify what the problem is.

**Comment by Jw-Jn** [comment]
Sure,
![image](https://user-images.githubusercontent.com/43404438/49617664-6e77e180-f96a-11e8-8bf2-ce73de13be35.png)[](url)

**Comment by otenim** [comment]
Thanks for your long patience.
We finally succeeded to reproduce the paper result on CelebA dataset.
Please just try the new weights we are sharing on our repository.
I hope that you see the improvement.
